Modules/Relationships/src/Relationships.Domain/Aggregates/Relationships/Relationship.cs

```diff
@@ -182,6 +182,37 @@ private void EnsureThereIsNoOpenReactivationRequest()
             throw new DomainException(DomainErrors.CannotRequestReactivationWhenThereIsAnOpenReactivationRequest());
     }
 
+    public void AcceptReactivation(IdentityAddress activeIdentity, DeviceId activeDevice)
+    {
+        EnsureOpenRevivalRequestExists();
+        EnsureRelationshipReactivationRequestIsAddressedToSelf(activeIdentity);
+
+        Status = RelationshipStatus.Active;
+
+        var auditLogEntry = new RelationshipAuditLogEntry(
+            RelationshipAuditLogEntryReason.AcceptanceOfReactivation,
+            RelationshipStatus.Terminated,
+            RelationshipStatus.Active,
+            activeIdentity,
+            activeDevice
+        );
+        AuditLog.Add(auditLogEntry);
+    }
+
+    private void EnsureOpenRevivalRequestExists()
+    {
+        if (AuditLog.Last().Reason != RelationshipAuditLogEntryReason.ReactivationRequested)
+        {
+            throw new DomainException(DomainErrors.CannotAcceptOrRejectRelationshipRevivalIfNoRequestToDoSoHasBeenMade());
+        }
+    }
+
+    private void EnsureRelationshipReactivationRequestIsAddressedToSelf(IdentityAddress activeIdentity)
+    {
+        if (To != activeIdentity)
```

Comment:
Here you check for the `To` property of the Relationship. That's not what you want

Reply:
done